src/main/java/edu/kis/powp/jobs2d/features/DriverFeature.java

```diff
@@ -35,10 +38,20 @@ public static void addDriver(String name, Job2dDriver driver) {
 		app.addComponentMenuElement(DriverFeature.class, name, listener);
 	}
 
+
+
+	@Override
+	public void update(Observable o, Object arg) {
+		app.updateInfo(driverManager.getCurrentDriver().toString());
```

Comment:
It is already implemented in DriverFeature.updateDriverInfo(). Simply delegate this task.